libs/blocks/code/code.js

```diff
@@ -0,0 +1,44 @@
+import * as utils from '../../utils/utils.js';
+
+export const HIGHLIGHT_JS = 'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/highlight.min.js';
```

Comment:
Let's store this in libs/deps

Reply:
I've copied over JS to libs/deps. I've also refactored theme support to light/dark variants and embedded that CSS instead.